Source/FakeItEasy/Core/FakeWrapperConfigurator.cs

```diff
@@ -47,6 +47,13 @@ public IFakeOptionsBuilder<T> WithAdditionalAttributes(IEnumerable<CustomAttribu
 
         public IFakeOptionsBuilder<T> Implements(Type interfaceType)
         {
+            Guard.AgainstNull(interfaceType, "interfaceType");
+
+            if (!interfaceType.IsInterface)
+            {
+                throw new ArgumentException("The specified type is not an interface");

```

Comment:
Oh, I see. I had always took the bit inside the `<exception>` element as a general description of the condition that would cause a condition, not a copy of the message that the exception would have.
I think the XML docs are fine as is, but unless you think there's no value in including the passed-in type in the exception message, perhaps update the message?

> According to `IFakeOptionsBuilder.Implements<TInterface>()`, that implementation should also throw for non-interfaces. I don't have the code to hand, but do you want the same fix applied to that (if required)?

(Note that I edited your comment above to put back ticks around  `IFakeOptionsBuilder.Implements<TInterface>()`. Without them, GitHub/HTML eat the `<TInterface>` and it looks like just IFakeOptionsBuilder.Implements().)
I think you propose to update those docs as well, if the other docs had required an update. That's generous, but I think both can stay as-is.